statannot/statannot.py

```diff
@@ -78,6 +81,17 @@ def stat_test(
             u_stat,
             pval,
         )
+    elif test == 'anova':
```

Comment:
Most tests (not t-test_something) are capitalized, so I'd suggest `Anova` for consistency